SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security policy
```

Comment:
Looks like we're checking `*.md` in some legacy client tests. As it's a Github-purpose -> can we move this file to `.github/`? This should fix the tests and still be recognized by GH.

Reply:
done